### PR TITLE
fix: skeleton loader issue in settings/api-keys

### LINKED ISSUE #4585: Skeleton loader for settings/api-keys empty state looks wrong
### Issue Summary

That's how it currently looks: 
Screenshot 2022-09-18 at 16 33 48

But it should look the same as the empty state of settings/webhooks

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #4585

https://www.loom.com/share/0a3d932cf17c49fab70aee33c47e25d9

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: apps/web/pages/v2/settings/developer/api-keys.tsx [hunk 1]
```diff
@@ -45,10 +45,10 @@ const ApiKeysView = () => {
         <>
           {isLoading && <SkeletonLoader />}
           <div>
-            {data?.length ? (
+            {isLoading && data === undefined ? null : !isLoading && data && data.length > 0 ? (
```
Comment (emrysal) [inline apps/web/pages/v2/settings/developer/api-keys.tsx:48]:
```suggestion
            {isLoading && !!data?.length ? (
```

Comment (hariombalhara) [inline apps/web/pages/v2/settings/developer/api-keys.tsx:48]:
```suggestion
            {isLoading ? null : data?.length ? (
```

**Review by hariombalhara — DISMISSED** [review]

File: apps/web/pages/v2/settings/developer/api-keys.tsx (hunk 1 above)
Comment (hariombalhara) [reply apps/web/pages/v2/settings/developer/api-keys.tsx:48]: It can be simplified as above.

Comment (G3root) [reply apps/web/pages/v2/settings/developer/api-keys.tsx:48]: that's better, i'll push the requested changes.

**Review by emrysal — APPROVED** [review]

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**